quipucords/quipucords/celery.py

```diff
@@ -16,4 +18,20 @@
 app.config_from_object("django.conf:settings", namespace="CELERY")
 
 # Load task modules from all registered Django apps.
-app.autodiscover_tasks()
+task_packages = ["scanner", "scanner.satellite.five", "scanner.satellite.six"]
```

Comment:
why are we limiting to those only?

Reply:
It's the other way around. Because of the way quipucords is currently structured, the default `autodiscover_tasks` _isn't actually discovering_ all the tasks. So, I have to explicitly define these module paths for now. Eventually I hope to make this better and bring us back to using defaults.